### ISSUE TITLE
BibTeX output is malformed

### ISSUE THREAD

**Issue by crism** [pkp/xmlps#22]
Running the EEG Comic Sans document through the unit tests, progressively, produces BibTeX of this form:
``` bibtex
@Book{RNiedermeyer,Silva,1993,
author="Niedermeyer, E.
and Silva, da Lopes F. H.",
title="Electroencephalography: Basic principles, clinical applications and related fields, 3rd edition",
year="1993",
publisher="Decker, Hamilton, Canada.",
address="Lippincott, Williams"
}
```
The ID must be a single token (see http://www.bibtex.org/Format/); Pandoc later chokes on this.

**Comment by axfelix** [comment]
Ah, this is what you meant about IDs blowing up. You're right -- that shouldn't be happening. We're definitely getting valid pandoc output on most of the documents I've tested but it's very possible that it's been blowing up in this way on some of them.

**Comment by kaschioudi** [comment]
@axfelix: Could you please point me to document that I can use to test this?

**Comment by axfelix** [comment]
[eeg_comicsans.docx](https://github.com/pkp/xmlps/files/61556/eeg_comicsans.docx)

yup, sorry for the delay -- that output was reported from this document in our old smaller corpus.

**Comment by kaschioudi** [comment]
@crism : what do you mean by "Pandoc later chokes on this.". My patch transforms "RNiedermeyer,Silva,1993" to "RNiedermeyerSilva1993" and everything else, including Pandoc, seems to work fine.

**Comment by axfelix** [comment]
Resolved along with #42
